### DISCUSSION TITLE
Docker LXC container

### DISCUSSION THREAD

**Discussion by Mrdindon** [tteck/Proxmox#1178] [Help Needed]
Good day,
I see that the container is deployed with the FUSE feature enabled in the options.
That seems to make an issue when backing up the the container with PBS (backup stays stuck at "create storage snapshot 'vzdump' ")

Not being familiar with FUSE anyhow, can this feature be disabled safely for this container ?

**Comment by VNRARA** [comment]
Same question here. Tried to research but not a real conclusive answer. I have mine turned off for now.
